Godeps/LICENSES

```diff
@@ -2542,237 +2542,6 @@ THE SOFTWARE.
 ================================================================================
 
 
-================================================================================
-= vendor/github.com/bazelbuild/bazel-gazelle licensed under: =
```

Comment:
I suppose we may need to be more diligent about licenses for things under hack/tools now?

Reply:
hmm… might be good to get an ack on what license requirements are for code we don't ship or link to, but make use of in dev tools/scripts. paging @kubernetes/licensing ...

Reply:
right

Reply:
will have to file a issue to follow up on this

Reply:
i meant we'll have lower visibility into e.g. changes, reviewers need to actually make a point to check these versus just looking at this file

Reply:
@kubernetes/dep-approvers

Reply:
@BenTheElder yep

Reply:
added support for a separate LICENSES file under hack/tools (which is updated by the update script and verified by the verify script)